chrome/content/zotero/preferences/preferences_general.xhtml

```diff
@@ -160,48 +160,36 @@
 		<groupbox>
 			<label><html:h2>&zotero.preferences.prefpane.locate;</html:h2></label>
 			
-			<hbox>
+			<hbox align="center">
+				<label control="openurl-menu" data-l10n-id="preferences-general-locate-resolver"></label>
 				<menulist id="openurl-menu" native="true">
 					<menupopup
 							id="openurl-primary-popup"
 							onpopupshowing="Zotero_Preferences.General.handleOpenURLPopupShowing(event)"
 							oncommand="Zotero_Preferences.General.handleOpenURLSelected(event)">
-						<menuitem label="&zotero.preferences.openurl.choose;"/>
+						<menuitem label="&zotero.preferences.openurl.custom;"/>
```

Comment:
Right now this shows as "Custom…" until you click on the drop-down, at which point it becomes "Custom". We should use just "Custom" (either added to Fluent as a general string and retrieved in `updateOpenURLResolversMenu` or just assigned from the existing properties string on load) and get rid of this .dtd string (English only).

Reply:
Gotcha. I went with using existing `general.custom` string as there is a step of setting the placeholder when the general preferences pane is initialized, so it's just a small tweak to that step.